openapi/components/schemas/Order.yaml

```
@@ -0,0 +1,203 @@
+type: object
+required:
+  - customerId
+  - websiteId
+  - items
+properties:
+  id:
+    type: string
+    description: ID of the order.
+    readOnly: true
+    maxLength: 50
+    example: ord_01GYJPRKHBD6ZYHH897QCJMBS4
+  organizationId:
+    readOnly: true
+    allOf:
+      - $ref: ./OrganizationId.yaml
+  customerId:
+    $ref: ./CustomerId.yaml
+  websiteId:
+    $ref: ./WebsiteId.yaml
+  activationInvoiceId:
+    type: string
+    description: ID of the activation invoice.
+    readOnly: true
+    maxLength: 50
+    example: in_0YVF9605RKC62BP14NE2R7V2XT
+  recentInvoiceId:
+    description: |-
+      ID of the most recently issued invoice.
+      The invoice might not be `paid` yet.
+    readOnly: true
+    type:
+      - 'string'
+      - 'null'
+    maxLength: 50
+    example: in_0YVF9605RKC62BP14NE2R7V2XT
+  upcomingInvoiceId:
+    type: string
+    description: ID of the upcoming invoice.
+    readOnly: true
+    maxLength: 50
+    example: in_0YVF9605RKC62BP14NE2R7V2XT
+  status:
+    description: |-
+      Temporary billing status of the most recent invoice.
+      This value may help you to determine if you should change the service status of the service,
+      such as suspending the service.
+    type: string
+    readOnly: true
+    enum:
+      - draft
+      - unpaid
+      - past-due
+      - abandoned
+      - paid
+      - voided
+      - refunded
+      - disputed
+      - partially-refunded
+      - partially-paid
+  items:
+    type: array
+    description: |-
+      Items in the order.
+
+      Note: Currently, this collection should contain no more than 1 element.
+    items:
+      type: object
+      description: Order entry details.
+      discriminator:
+        propertyName: type
+        mapping:
+          subscription: ./OrderSubscription.yaml
+          one-time-sale: ./OrderOneTimeSale.yaml
+      anyOf:
+        - $ref: ./OrderSubscription.yaml
+        - $ref: ./OrderOneTimeSale.yaml
+  invoiceIssueTimeShift:
+    description: |-
+      Invoice issue time shift interval in ISO-8601 format.
+
+      This field adjusts the issue date and time of renewal invoices for the order.
+
+      This interval value supports the `-` symbol.
+    type:
+      - 'string'
+      - 'null'
+    example: +P00Y00M00DT00H00M00S
+  invoiceDueTimeShift:
+    description: |-
+      Invoice due time shift interval in ISO-8601 format.
+
+      This field adjusts when the payment is due for the order.
```

suggestion: not sure if I have changed the meaning (I was confused by this line when I first read it)
```suggestion
      Use this field to adjust when payment is due for the order.
```